client/my-sites/stats/stats-strings.js

```diff
@@ -86,14 +86,14 @@ export default function() {
 		} ),
 	};
 
-	statsStrings.podcastdownloads = {
-		title: translate( 'Podcasts', { context: 'Stats: title of module' } ),
-		item: translate( 'Episodes', { context: 'Stats: module row header for podcast.' } ),
+	statsStrings.filedownloads = {
+		title: translate( 'File Downloads', { context: 'Stats: title of module' } ),
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?